src/components/single-line-diagram-viewer/single-line-diagram-viewer.ts

```diff
@@ -485,7 +485,7 @@ export class SingleLineDiagramViewer {
                         }
                         const switchId = aSwitch.equipmentId;
                         const open = aSwitch.open;
-                        this.onBreakerCallback?.(switchId, !open, event.currentTarget);
```

Comment:
isn't the definition of `SvgInHtml` missing? The error can be seen in the CI (when building), but it doesn't fail I don't know why.

Reply:
true i don't know why XD
the type was not correct, i changed it to SVGElement